pages/profile/[username].tsx

```diff
@@ -14,50 +15,79 @@ import Head from "next/head"
 
 import Lottie from "react-lottie"
 import animationData from "../../lottie/sad.json"
-const defaultOptions = {
+import {ProfileSerializer} from "types/backend"
+const defaultOptions: {
+    loop: boolean
+    autoplay: boolean
+    animationData: any | null
+} = {
     loop: true,
     autoplay: true,
     animationData: animationData,
     // rendererSettings: {
     // 	preserveAspectRatio: 'xMidYMid slice'
     // }
 }
+export interface Auth {
+    token?: string | null
+    profile?: any | null
+    loading?: boolean
+}
+export interface RUser {
```

Comment:
Some better name? Maybe `RequestUserType` or something more explicit of what it is for.